explorer/src/components/Consensus/Extrinsic/ExtrinsicDetailsEventList.tsx

```diff
@@ -30,12 +28,12 @@ type Row = EventsByExtrinsicIdQuery['consensus_events'][number]
 
 type Props = {
```

Comment:
we can also update this Generic Props name to something specific 

`Props` ---> `ExtrinsicDetailsEventListProps`